<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportView</class>
 <widget class="QDialog" name="ImportView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Импорт</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="pathEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="choosePathBtn">
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="excelTemplateBtn">
       <property name="text">
        <string>Загрузить шаблон Excel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importBtn">
       <property name="text">
        <string>Импорт</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
